src/Api/Fixtures/MediaFixtures.php

```diff
@@ -62,15 +62,15 @@ public function __construct()
     }
 
     /**
-     * Get media
+     * Get media.
      *
      * @param $id
-     *   Media ID to fetch.
+     *   Media ID to fetch
      *
-     * @return Media|null
-     *   Media object or null if not found.
+     * @return media|null
+     *                    Media object or null if not found
```

Comment:
Do we want this rule?

Reply:
NO

Reply:
I don't like it but I also don't want to deviate from https://symfony.com/doc/current/contributing/code/standards.html

Reply:
Then it should be on a single line

Reply:
Done